unime/src/i18n/nl-NL/index.ts

```diff
@@ -189,26 +191,59 @@ const nl = {
       DATA: 'Data',
       BADGES: 'Badges',
     },
+    CREDENTIAL: {
+      NAVBAR_TITLE: 'Credential informatie',
+    },
+    BADGE: {
+      NAVBAR_TITLE: 'Badge informatie',
+      DESCRIPTION: 'Description',
+      METADATA: 'Metadata',
+    },
     EMPTY_CREDENTIALS: {
       TITLE: 'Momenteel is het hier wat rustig',
       SUBTITLE: 'Wat denk je ervan om nieuwe credentials toe te voegen om je digitale "me" te starten?',
     },
     FAVORITES: 'Mijn favorieten',
   },
   ACTIVITY: {
+    BOTTOM_NAVIGATION_TITLE: 'Activiteiten',
     NAVBAR_TITLE: 'Verbonden',
     TABS: {
       CONNECTIONS: 'Verbindingen',
       TIMELINE: 'Tijdlijn',
     },
   },
+  SCAN: {
+    BOTTOM_NAVIGATION_TITLE: 'Scan',
+    TITLE_1: 'Scan een',
+    TITLE_2: 'QR Code',
+    SUBTITLE: 'Breng een QR-code in beeld op dit scherm om een interactie te starten.',
+    PERMISSION_1: 'Geen recht om toegang',
+    PERMISSION_2: 'te krijgen tot de camera',
+    CONNECTION_OFFER: {
+      NAVBAR_TITLE: 'Credential Aanbod',
+      DESCRIPTION: 'biedt u de volgende credentials',
+    },
+    CONNECTION_REQUEST: {
+      NAVBAR_TITLE: 'Credential Aanvraag',
+      TITLE_1: 'Nieuwe verbinding',
```

Comment:
"Verbinding" to me sounds more like some connection to a 'machine' or 'thing', whereas "Connectie" sounds to me more like a connection to a person or an organization. So I would rather use 'Nieuwe connectie' here instead.